src/trials.jl

```diff
@@ -98,12 +136,16 @@ mutable struct TrialEstimate
     gctime::Float64
     memory::Int
     allocs::Int
+    linux_perf_stats::Union{LinuxPerf.Stats,Nothing}
 end
 
 function TrialEstimate(trial::Trial, t, gct)
-    return TrialEstimate(params(trial), t, gct, memory(trial), allocs(trial))
+    return TrialEstimate(
+        params(trial), t, gct, memory(trial), allocs(trial), trial.linux_perf_stats
+    )
 end
 
+# Should we compare linux_perf_stats here?
 function Base.:(==)(a::TrialEstimate, b::TrialEstimate)
```

Comment:
Should we check if `linux_perf_stats` are the same here? Given it's essentially a hidden field right now, I'd say you could argue we shouldn't.